doc/source/user_guide/basics.rst

```diff
@@ -2050,33 +2050,35 @@ The following table lists all of pandas extension types. For methods requiring `
-|                                                 |                           |                    |                               | ``'Sparse[float]'``                    |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`intervals <advanced.intervalindex>`       | :class:`IntervalDtype`    | :class:`Interval`  | :class:`arrays.IntervalArray` | ``'interval'``, ``'Interval'``,        |
-|                                                 |                           |                    |                               | ``'Interval[<numpy_dtype>]'``,         |
-|                                                 |                           |                    |                               | ``'Interval[datetime64[ns, <tz>]]'``,  |
-|                                                 |                           |                    |                               | ``'Interval[timedelta64[<freq>]]'``    |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`nullable integer <integer_na>`            | :class:`Int64Dtype`, ...  | (none)             | :class:`arrays.IntegerArray`  | ``'Int8'``, ``'Int16'``, ``'Int32'``,  |
-|                                                 |                           |                    |                               | ``'Int64'``, ``'UInt8'``, ``'UInt16'``,|
-|                                                 |                           |                    |                               | ``'UInt32'``, ``'UInt64'``             |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`Strings <text>`                           | :class:`StringDtype`      | :class:`str`       | :class:`arrays.StringArray`   | ``'string'``                           |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
-| :ref:`Boolean (with NA) <api.arrays.bool>`      | :class:`BooleanDtype`     | :class:`bool`      | :class:`arrays.BooleanArray`  | ``'boolean'``                          |
-+-------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
++--------------------------------------------------+---------------------------+--------------------+-------------------------------+----------------------------------------+
+| Kind of Data                                     | Data Type                 | Scalar             | Array                         | String Aliases                         |
```

Comment:
looks like this has gone forward by one space 😄 any chance you could bring it back to where it was, to minimise the diff?

Reply:
Thank you, @MarcoGorelli . I was wondering why my diff was so unreadable. I deleted one space, now it looks nicer 😄.

Also, I reverted line 73 as you suggested.